tests/UI/tsconfig.json

```diff
@@ -19,6 +19,8 @@
       "@utils/*": ["utils/*"],
       "@tools/*": ["tools/*"]
     },
-    "typeRoots": ["types", "node_modules/@types"]
+    "typeRoots": ["types", "node_modules/@types"],
+    "outDir": "dist",
+    "skipLibCheck": true
```

Comment:
```suggestion
    "typeRoots": ["types", "node_modules/@types"],
    "exclude": ["node_modules"],
    "noEmit": true
```

Reply:
Or not : 
![image](https://user-images.githubusercontent.com/1533248/209653601-22b137ef-9d93-4d54-952c-053a8dab74b5.png)

Reply:
But the noEmit is :ok_hand: